scalafix-core/shared/src/main/scala/scalafix/internal/rule/RuleCtxImpl.scala

```diff
@@ -48,19 +50,18 @@ case class RuleCtxImpl(tree: Tree, config: ScalafixConfig) extends RuleCtx {
     logger.elem(values: _*)
   }
 
-  def printLintMessage(msg: LintMessage, owner: RuleName): Unit = {
-    val key = msg.category.key(owner)
-    if (config.lint.ignore.matches(key)) ()
-    else {
-      val category = config.lint
-        .getConfiguredSeverity(key)
-        .getOrElse(msg.category.severity)
-      config.lint.reporter.handleMessage(
-        msg.format(owner, config.lint.explain),
-        msg.position,
-        category.toSeverity
-      )
-    }
+  def printLintMessage(msg: LintMessage): Unit = {
+    val key = msg.category.id
+
+    val category = config.lint
```

Comment:
We may want to move this into `LintCategory.withConfig(LintConfig): LintCategory`

Reply:
We can open an issue and fix it in a separate PR.